src/main/java/io/jenkins/plugins/conventionalcommits/ConventionalCommits.java

```diff
@@ -27,13 +27,18 @@ public Version nextVersion(Version in, List<String> commits) {
     }
 
     private boolean breakingChangeFooter(String commit){
-        int startIndex = commit.lastIndexOf("\n");
 
-        String result = commit;
+        boolean result = false;
+        String[] lines = commit.split("[\\r\\n]+");
 
-        if(startIndex!=-1 && startIndex!= commit.length()){
-            result = commit.substring(startIndex+1);
+        for(String line: lines){
+            if (line.startsWith("BREAKING CHANGE") || line.startsWith("BREAKING-CHANGE")) {
```

Comment:
It would be better to look for "BREAKING CHANGE:" instead of "BREAKING CHANGE" otherwise the following example would trigger breaking change

"BREAKING CHANGE IS NOT CONSIDERED IN THIS COMMIT" 

According to documentation the expected footer BREAKING CHANGE: <description>

Reply:
Oh right! My bad, Thank you so much for catching that! 😄